example.d

```diff
@@ -160,6 +161,86 @@ class TestingThisAndThat
         testResult(false);
     }
 
+    // enabled by condition test function
```

Comment:
Change the expected outcome in the README accordingly: more than one skip!

    (When you get four failures, one error, and one skip, everything works fine.)

Reply:
fixed